web/src/compose.ts

```diff
@@ -64,9 +96,10 @@ export function show_preview_area() {
     render_preview_area();
 }
 
-export function render_preview_area() {
-    const $compose_textarea = $("textarea#compose-textarea");
+export function render_preview_area(): void {
+    const $compose_textarea = $<HTMLTextAreaElement>("textarea#compose-textarea");
     const content = $compose_textarea.val();
+    assert(content !== undefined);
```

Comment:
Probably this should be using `compose_state.message_content()` and then you wouldn't need the `assert`.